File: blink.py
import RPi.GPIO as GPIO
import time

PIN = 17

GPIO.setmode(GPIO.BCM)
GPIO.setup(PIN, GPIO.OUT)

for i in range(15):
    GPIO.output(PIN, GPIO.HIGH)
    time.sleep(0.5)
    GPIO.output(PIN, GPIO.LOW)
    time.sleep(0.5)

GPIO.cleanup()

File: blink.js
let gpio = require('rpi-gpio');

const LED_PIN = 40;
let ledOn = true;

gpio.setup(LED_PIN, gpio.DIR_OUT, () => {
    setInterval(() => {
        if (ledOn) {
            gpio.write(LED_PIN, false);
            ledOn = false;
            console.log("fugafuga");
        }
        else {
            gpio.write(LED_PIN, true);
            ledOn = true;
            console.log("hogehoge");
        }
    }, 500);
});

